coursebuilder/modules/explorer/_static/js/course.js

```diff
@@ -20,13 +20,27 @@ $(document).ready(function documentReady() {
     var sidebarWidth = sidebarToggle ? $('.course-accordion').width() : 0;
 
     $('.course-accordion').toggleClass('hide-sidenav');
-    $('.course-card').animate({'margin-left': sidebarWidth / 2 + 'px'}, 300);
-    $('.course-accordion').animate({'margin-left': '-' + sidebarWidth + 'px'}, 300);
+    $('.course-card').animate({ 'margin-left': sidebarWidth / 2 + 'px' }, 300);
+    $('.course-accordion').animate({ 'margin-left': '-' + sidebarWidth + 'px' }, 300);
   });
 
   $('.course-accordion .collapsible').clone().appendTo('#courseMobileSidebar');
   $('.collapsible').collapsible();
   $('body').on('click', '#closeSideNav', function () {
     $('.button-collapse').sideNav('hide');
   });
+
+  $('.course-content').on('click', function (event) {
+    var courseCarousel = $(event.target).closest('.owl-nav');
```

Comment:
@andela-johia  research on how .closest works in relation the DOM.

Reply:
@chegejames Alright I will look more into it. Thanks

Reply:
@chegejames For better user experience, when a user clicks within the carousel navigation panel, an option to use the arrow keys is activated. But if the user clicks outside this panel for example a textarea within the carousel, using the arrow keys to navigate the carousel is deactivated. 
The .closest function helps us achieve the above by returning the first parent if it is within the navigation panel or else it returns a length of 0. The arrow keys navigation is wrapped in a conditional statement to ensure that the user clicks within the navigation panel before it is enabled.